README.md

```diff
@@ -47,15 +100,14 @@ make_idl_files.py MTM1M3 MTM1M3TS MTMount MTVMS
 
 ## SALComm
 
-Heart of the application is SALComm. The module links ts_salobj callbacks with
```

Comment:
MD really doesn't need to format text on sentences. This is formated in vim, keeping 70 char wide terminal. Please keep the original formatting.

Reply:
Updated in the commit: 535c3c1. Thanks!